aspnetcore/blazor/debug.md

```diff
@@ -36,11 +36,11 @@ Development of further debugging scenarios is an on-going focus of the engineeri
 
 To debug a Blazor client-side app in Chrome:
 
-* Build a Blazor app in `Debug` configuration (the default for unpublished apps).
-* Run the Blazor app in Chrome (version 70 or later).
-* With the keyboard focus on the app (not in the developer tools panel, which you should probably close for a less confusing debugging experience), select the following Blazor-specific keyboard shortcut:
-  * `Shift+Alt+D` on Windows/Linux
-  * `Shift+Cmd+D` on macOS
+1. Run a Blazor client-side app in `Debug` configuration. Pass the `--configuration Debug` option to the [dotnet run](/dotnet/core/tools/dotnet-run) command: `dotnet run --configuration Debug`.
+1. Access the app in Chrome (version 70 or later) or with a Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com)).
```

Comment:
Remove that line. Why would you need it? How else are you going to access it under section?

Reply:
What? I don't follow. You have to use Chrome 70 or later or Edge preview.

Reply:
Shouldn't that be a prereq then, listed earlier. I wouldn't be happy to find out only those versions work at this point.

Reply:
ok ... sounds good.

Reply:
To debug a Blazor client-side app in Chrome 70 or higher or Edge whaterver: